### ISSUE TITLE
Expose options via CLI

### ISSUE THREAD

**Issue by KlausJokisuo** [jnross/Bluetility#15]
Would it be possible to use this wonderful application via CLI?

**Comment by jnross** [comment]
That would certainly be possible to build, but not something I'm planning to do anytime soon.  Pull requests welcome though!

**Comment by RobT7366** [comment]
Yeah, I'd LOVE a command line interface. i don't have the Swift skills to jump in and do this, but I'd happily share some work that I'm doing to RE IOT devices to use MacOS for better control.

**Comment by jnross** [comment]
Perhaps if you all could give some examples of the interface you're looking for, that might inspire me or someone else to get working on this...  

When I think CLI, I think a bunch of short interactions with discrete inputs and outputs.  I suppose a fully-interactive CLI with a REPL might make sense for some as well.  If you give examples of what commands might look like, and what the expected output should look like, that would help.
